sub/source/docs/introduction.rst

```diff
@@ -4,15 +4,36 @@
 Introduction to Sub
 ===================
 
-**UNDER CONSTRUCTION**
+Sub is an advanced open-source autopilot system for submersible ROVs (Remote Operated Vehicle) that supports multiple vehicle configurations. It offers a variety of operating modes from fully manual to fully autonomous.
 
+.. image:: ../images/sub-system.jpg
+    :target: ../_images/sub-system.jpg
 
 
-Vehicle Configuration Types
-===========================
-
+Vehicle Configuration Options
+=============================
 
+* Sub supports both brushed and brushless, uni-directional or bi-directional propulsion motors/thrusters
+* 3/4/5 or 6 thruster configurations controlling up to all 6 degrees of vehicular motion and axial rotation
+* Pilot control and telemetry via cable tether to a Ground Control Station (GCS) using joystick, button, and/or keyboard commands, or
+* Pilot control and telemetry via radio control to tethered surface buoy
+* Position and Depth Control using optional depth sensor, GPS while surfaced, Sonars, or Shore based acoustic beacon locators
```

Comment:
the acoustic beacons are often on a support vessel, which I think "Shore based" is not capturing

Reply:
will modify